templates/zones_list.html

```diff
@@ -90,6 +90,7 @@ <h2>{{ _("France") }}</h2>
 <p>&bull;<a href="/annot"> Annot</a></p>
 <p>&bull;<a href="/orlu"> Orlu</a></p>
 <p>&bull;<a href="/laramade"> Laramade</a></p>
+<p>&bull;<a href="/pont_de_camps"> Pont de Camps</a></p>
```

Comment:
Adding zones to list is no longer required, running `python update_pages_and_templates.py` generates the required pages and templates to display the new zone as well as including it in the lists and maps.

Reply:
How can you control then the position in the list?